scipy/stats/_continuous_distns.py

```diff
@@ -517,15 +517,19 @@ def _entropy(self):
 
 class FitDataError(ValueError):
     # This exception is raised by, for example, beta_gen.fit when both floc
-    # and fscale  are fixed and there are values in the data not in the open
-    # interval (floc, floc+fscale).
-    def __init__(self, distr, lower, upper):
-        self.args = (
-            "Invalid values in `data`.  Maximum likelihood "
-            "estimation with {distr!r} requires that {lower!r} < x "
-            "< {upper!r} for each x in `data`.".format(
-                distr=distr, lower=lower, upper=upper),
-        )
+    # and fscale are fixed and there are values in the data not in the open
+    # interval (floc, floc+fscale), or with a custom message.
+    def __init__(self, distr, lower, upper, **kwds):
+        msg = kwds.pop('msg', None)
```

Comment:
Just curious - why not a `msg` keyword argument with `None` default value?

Reply:
Good question. I'm not sure I had a particular reason to do this other than that I already had **kwargs on the mind. I think `msg=None` is cleaner though. And, like you previously mentioned, we could alos give the other arguments default values they don't have to be required anymore.